source/features/view-markdown-source.tsx

```diff
@@ -1,129 +1,30 @@
-	const randomKnownFile = 'https://github.com/sindresorhus/refined-github/blob/b1229bbaeb8cf071f0711bc2ed1b40dd96cd7a05/.editorconfig';
-
-	// Fetch background.js due to CORB policies
-	const html = await browser.runtime.sendMessage({fetch: randomKnownFile});
-	const blobToolbar = domify(html).querySelector('.BlobToolbar')!;
-	select('a#js-view-git-blame', blobToolbar)!.href = new GitHubURL(location.href).assign({route: 'blame'}).href;
-	select('a#js-new-issue', blobToolbar)!.href = buildRepoURL('issues/new');
-	return blobToolbar;
-});
-
-const buttonBodyMap = new WeakMap<Element, Element | Promise<Element>>();
-
-async function fetchSource(): Promise<Element> {
-	const path = location.pathname.replace(/((?:[^/]+\/){2})(blob)/, '$1blame');
-	const dom = await fetchDom(path, '.blob-wrapper');
-	dom!.classList.add('rgh-markdown-source');
-	return dom!;
-}
-
-// Hide tooltip after click, it’s shown on :focus
-function blurButton(button: HTMLElement): void {
-	if (button === document.activeElement) {
-		button.blur();
-	}
-}
-
-function dispatchEvent(element: HTMLElement, type: keyof GlobalEventHandlersEventMap): void {
-	element.dispatchEvent(new CustomEvent(type, {bubbles: true}));
-}
-
-/*
-The dom of each version is stored on each button.
-This acts as an auto-discarded cache without globals, timers, etc.
-It should also work clicks on buttons sooner than the page loads.
-*/
-async function showSource(): Promise<void> {
-	const sourceButton = select('button.rgh-md-source')!;
-	const renderedButton = select('button.rgh-md-rendered')!;
-
-	sourceButton.disabled = true;
-
-	const source = buttonBodyMap.get(sourceButton) ?? fetchSource();
-	const rendered = await buttonBodyMap.get(renderedButton) ?? select('.blob.js-code-block-container')!;
-
-	buttonBodyMap.set(sourceButton, source);
-	buttonBodyMap.set(renderedButton, rendered);
-
-	rendered.replaceWith(await source);
-
-	sourceButton.disabled = false;
-
-	sourceButton.classList.add('selected');
-	renderedButton.classList.remove('selected');
-	blurButton(sourceButton);
-	dispatchEvent(sourceButton, 'rgh:view-markdown-source');
-
-	(await source).before(await lineActions());
-}
-
-async function showRendered(): Promise<void> {
-	const sourceButton = select('button.rgh-md-source')!;
-	const renderedButton = select('button.rgh-md-rendered')!;
-
-	renderedButton.disabled = true;
-
-	(await buttonBodyMap.get(sourceButton))!.replaceWith(await buttonBodyMap.get(renderedButton)!);
-
-	renderedButton.disabled = false;
-
-	sourceButton.classList.remove('selected');
-	renderedButton.classList.add('selected');
-	blurButton(renderedButton);
-	dispatchEvent(sourceButton, 'rgh:view-markdown-rendered');
-
-	(await lineActions()).remove();
-}
 
 async function init(): Promise<void> {
-	delegate(document, '.rgh-md-source:not(.selected)', 'click', showSource);
-	delegate(document, '.rgh-md-rendered:not(.selected)', 'click', showRendered);
-
+	const URLWithoutSearchParameters = window.location.href.split('?')[0];
```

Comment:
This should use `.searchParams.remove('plain')` instead of dropping all search parameters. And the other one should use `.add('plain', '1')`